tests/test_celery.py

```diff
@@ -0,0 +1,23 @@
+import re
+
+from merlin.celery import app
+
+
+def test_broker_url():
+    """
+    Ensure the celery application 'broker_url' roughly matches the required pattern.
+    """
+    if app.conf.broker_url:
+        assert re.match(
+            r"amqps:\/\/\w+:.+@jackalope\.llnl\.gov:\d+\/\w+", app.conf.broker_url
+        )
+
+
+def test_result_backend():
```

Comment:
Having a redis backend is not required. I would eliminate these tests. There ias a bug anyone, using broker_url instead of backend.